actors/builtin/miner/miner_test.go

```diff
@@ -1952,44 +1919,39 @@ func TestWindowPost(t *testing.T) {
 			dlinfo = advanceDeadline(rt, actor, &cronConfig{})
 		}
 
-		// Now submit PoSt with all faults skipped
-		// Sectors should be penalized as an undeclared fault and unproven power won't be activated.
-		// Fee for skipped fault is undeclared fault fee, but it is split into the ongoing fault fee
-		// which is charged at next cron and the rest which is charged during submit PoSt.
-		undeclaredFee := actor.undeclaredFaultPenalty(infos)
-		declaredFee := actor.declaredFaultPenalty(infos)
-		faultFee := big.Sub(undeclaredFee, declaredFee)
-
+		// Now submit PoSt with all sectors skipped. Unproven power won't be activated.
```

Comment:
Maybe "PoSt fails to validate leaving power un-activated"? Note the `expectedPowerDelta` below could be anything and this test will still pass. `cfg` should probably just be `nil`.